packages/java/maven-plugin/src/main/java/com/vaadin/fusion/maven/ParserConfiguration.java

```diff
@@ -46,6 +50,10 @@ public boolean equals(Object other) {
             return Objects.equals(name, ((Plugin) other).name);
         }
 
+        public Optional<PluginConfiguration> getConfiguration() {
```

Comment:
why use Optional here?

Reply:
Just to note that it could be `null`. Should I remove it?

Reply:
In the code, I see there is `orElse(null)`, so probably doesn't need this optional wrapper in the first place.

Reply:
Ok, replaced with `@Nonnull` annotation for all non-nullable values